.dotnet/tests/OpenAI.Tests.csproj

```diff
@@ -1,9 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
     <!--Ignore XML doc comments on test types and members-->
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <LangVersion>latest</LangVersion>
+    <!--<SignAssembly>True</SignAssembly>
+    <AssemblyOriginatorKeyFile Include="..\src\OpenAI.snk" />-->
```

Comment:
Not sure if we want to sign the test project, or what approach we would want to take to do that, but if we do, we can comment out these lines (with some small modifications), and then uncomment the mock test of the StreamingChatUpdateCollection.